### PR TITLE
Provide support for running the app as container

### LINKED ISSUE #2: Build an isolated running environment
We should build an isolated environments instead of installing all dependencies, systems on local machine.

We'll build docker environment is made of two parts of pyreportcard web server and mongodb as database server

### PR DESCRIPTION
**Pull request by addianto**
This PR provides a mechanism to run the app locally using Docker container and `docker-compose`.

Besides making the app runnable in containerised environment, this PR also introduces the following changes:

- Reordered Flask app initialisation (see: https://github.com/mingrammer/pyreportcard/pull/35/commits/b207d004e74085e9067329f75c60307981eb2e94).
- Added `MONGO_URI` (see: https://github.com/mingrammer/pyreportcard/pull/35/commits/8d10b46d33a5abda6b972c1e92b440d99a8c84c7) in Config object. It was probably due to latest version of MongoDB driver used by Flask that now requires `MONGO_URI` property.
- Separated document analysers from running altogether with count-based analysers (see: https://github.com/mingrammer/pyreportcard/pull/35/commits/4262b9d21d8d6cb4cc7374e8ab5ce72a5fd0373a).

Resolves #2.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,7 +64,32 @@ class SecretConfig:
...
+```
+
+To run the app and MongoDB, build the app and run them using `docker-compose`.
+By default, the app will bind to port `5000` and can be accessed via `localhost`,
+e.g. `127.0.0.1:5000`. If the port conflicted with other running apps, feel free
+to modify the port binding in `docker-compose.yml` and re-run the app.
+
+```bash
+docker-compose build
+docker-compose up -d
```
Comment (mingrammer) [inline README.md]: You can do this with just one command `docker-compose up --build -d`

**Comment by mingrammer** [comment]
Is this in draft? :)

File: README.md (hunk 1 above)
Comment (addianto) [reply README.md]: Thanks for the insight. I'm fixing it now.

**Comment by addianto** [comment]
By the way, do I need to squash the commits? Or will GitHub squash them automatically?

**Comment by mingrammer** [comment]
I can squash when merging the PR :). I'll do!

**Review by mingrammer — APPROVED** [review]
LGTM

**Comment by mingrammer** [comment]
Thank you for contributing

**Pull request merged**